web/api/rest-api-design/Batching-Filtering-Pagination.md

```diff
@@ -0,0 +1,73 @@
+## Batching, Filtering, and Pagination
+
+### Filtering:
+
+- Topics covered: Elasticsearch and URL Parameters
+
+#### Elasticsearch:
+
+- Elasticsearch is a distributed RESTful search engine that is recommended for most filtering needs.
```

Comment:
I'm not sure I agree with this. For example, if you're already using Postgres in your app, it is very capable of filtering and might already fit your needs, which will prevent introducing Elasticsearch as another dependency to manage. Postgres can also do full-text search.